internal/backends/mcu/lib.rs

```diff
@@ -34,12 +34,17 @@ mod renderer;
 use lengths::*;
 
 /// The Pixel type of the backing store
-/// TODO: make it configurable
+#[cfg(not(feature = "stm32h735g"))]
 pub type TargetPixel = embedded_graphics::pixelcolor::Rgb565;
```

Comment:
I'm curious: TargetPixel defined as Rgb565 is disabled here for STM32H7 and inside the stm32h7 module it is defined as Rgb565.

I guess it should become configurable, but for now it seems that all backends use Rgb565 and we can just use this definition?

Reply:
I thought that the screen was RGB888 so at first i had it set to RGB888 in the stm backend, but it turns out that RGB888 shows garbage on the screen for some reason, so maybe the screen don't support it after all.